secretstores/tencentcloud/ssm/ssm.go

```diff
@@ -0,0 +1,189 @@
+/*
+Copyright 2021 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ssm
+
+import (
+	"context"
+	"errors"
+	"strconv"
+
+	"github.com/tencentcloud/tencentcloud-sdk-go/tencentcloud/common"
+	"github.com/tencentcloud/tencentcloud-sdk-go/tencentcloud/common/profile"
+	ssm "github.com/tencentcloud/tencentcloud-sdk-go/tencentcloud/ssm/v20190923"
+
+	"github.com/dapr/components-contrib/secretstores"
+	"github.com/dapr/kit/logger"
+)
+
+const (
+	SecretID         = "SecretID"
+	SecretKey        = "SecretKey"
+	Token            = "Token"
+	Region           = "Region"
+	VersionID        = "VersionID"
+	RequestID        = "RequestID"
+	ValueType        = "SecretValueType"
+	pageLimit uint64 = 100
+)
+
+type SecretValueType uint16
+
+var (
+	// TextSecretValue text secret.
+	TextSecretValue SecretValueType = 10
+	// BinarySecretValue binary secret.
+	BinarySecretValue SecretValueType = 20
+)
+
+type ssmClient interface {
+	ListSecretsWithContext(ctx context.Context, request *ssm.ListSecretsRequest) (*ssm.ListSecretsResponse, error)
+	GetSecretValueWithContext(ctx context.Context, request *ssm.GetSecretValueRequest) (*ssm.GetSecretValueResponse, error)
+}
+
+type ssmSecretStore struct {
+	client ssmClient
+	logger logger.Logger
+}
+
+var _ secretstores.SecretStore = (*ssmSecretStore)(nil)
```

Comment:
Nit: because `NewSSM` returns an interface now (secretstores.SecretStore), these type assertions aren't needed anymore

Reply:
Ok. I delete it.